src/mito2/src/compaction/picker.rs

```diff
@@ -38,7 +39,7 @@ pub trait Picker: Debug + Send + Sync + 'static {
 
 /// PickerOutput is the output of a [`Picker`].
 /// It contains the outputs of the compaction and the expired SST files.
-#[derive(Default, Clone, Debug)]
+#[derive(Default, Clone, Debug, Serialize, Deserialize)]
 pub struct PickerOutput {
     pub outputs: Vec<CompactionOutput>,
     pub expired_ssts: Vec<FileHandle>,
```

Comment:
Could we only hold `FileMeta` in `PickerOutput`?
```suggestion
    pub expired_ssts: Vec<FileMeta>,
```

Reply:
Let me dig deeper into the difference between `FileHandle` and `FileMeta`.

Reply:
I think it's hard to replace `FileMeta` with `FileHandle` in `PickerOutput` because the compaction needs some metadata that records in `FileHandle`. In addition to the `expired_ssts`, the `CompactionOutput` in `PickerOutput` also uses `FileHandle`. I want to try @WenyXu's advice for creating a new serializable struct, but it seems it just rewrites the `PickerOuput` by replacing all `FileHandle` with `FileMeta`(looks ugly...).

Reply:
Let me remove the `Serialize` from `PickerOuput`. I will rewrite the struct into a serialized version in the plugins.